src/Servers/Kestrel/shared/KnownHeaders.cs

```diff
@@ -359,6 +366,32 @@ static string AppendHPackSwitchSection(HPackGroup group)
         }
     }
 
+    static string AppendQPackSwitchSection(QPackGroup group)
+    {
+        var header = group.Header;
```

Comment:
Nit: This could be combined with `AppendHPackSwitchSection` by taking the `KnownHeader` directly. Maybe rename it to `Append